bitnami/postgresql-ha/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: postgresql-ha
-version: 2.3.1
+version: 2.3.2
```

Comment:
We should bump the minor version when adding a new feature/parameter to the chart